### PR TITLE
pip-29: Draft Batch Transactions

### PR DESCRIPTION
**Pull request by maxipaz**
Draft for a new PIP to define `Batch Transactions` in Pactus.

Should be discussed and get completed.

### CODE REVIEW

File: PIPs/pip-29.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+## Specification
+
+**Transaction Format:**
+
+- Utilize the `Flags` field to indicate a Batch Transaction with a new flag: `0x02`.
+- Modify Transfer Transactions to allow an array of receivers.
+- Support different transaction types in one batch by using the `Payload Data` field
+  (we should have an array of Payload Data).
+
+## Pros
```
Comment (b00f) [inline PIPs/pip-29.md:41]: We don't have this title (Header-2) in PIP-templates. 
Let's keep it consistent with others. You can check the PIP template here: [PIP Template](https://github.com/pactus-project/PIPs/blob/main/pip-template.md?plain=1)

File: PIPs/pip-29.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+- **Enhanced User Experience:** Simplifies the process for users with complex transaction needs.
+
+## Cons
+
+- **Complexity:** Increases the complexity of transaction validation and processing.
+- **Security:** Requires rigorous security measures to prevent batch transaction abuse.
+- **Redundancy:** If gas-less transactions are implemented, the primary benefit of batch transactions (cost saving) is diminished.
+- **Debugging Difficulties:** Troubleshooting failed transactions can become more complex.
+
+## Conclusion
```
Comment (b00f) [inline PIPs/pip-29.md:56]: This title also not available in PIP-template.

**Review by b00f — CHANGES_REQUESTED** [review]
@maxipaz Overall, it looks good. We can add it to the PIPs and start discussing it.

**Comment by Ja7ad** [comment]
@maxipaz Please fix request changes before merge.

**Pull request merged**